spec.html

```diff
@@ -5546,6 +5746,110 @@ <h1>CopyDataProperties ( _target_, _source_, _excludedItems_ )</h1>
         <p>The target passed in here is always a newly created object which is not directly accessible in case of an error being thrown.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-newprivatename" aoid="NewPrivateName">
+      <h1>NewPrivateName ( _description_ )</h1>
+      <p>The abstract operation NewPrivateName takes argument _description_ (a String). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: Type(_description_) is String.
+        1. Return a new unique Private Name whose [[Description]] value is _description_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-privateelementfind" aoid="PrivateElementFind">
+      <h1>PrivateElementFind ( _P_, _O_ )</h1>
+      <p>The abstract operation PrivateElementFind takes arguments _P_ (a Private Name) and _O_ (an Object). It performs the following steps when called:</p>
+      <emu-alg>
+        1. For each PrivateElement Record _entry_ of _O_.[[PrivateElements]], do
+          1. If _entry_.[[Key]] is _P_, return _entry_.
```

Comment:
```suggestion
        1. If _O_.[[PrivateElements]] contains a PrivateElement Record whose [[Key]] is _P_, then
          1. Return that PrivateElement Record.
```
this is the verbage used in ResolvePrivateIdentifier - i'm not sure which is better, but it seems like both should be consistent, since they're doing the same kind of find operation on a List.